### ISSUE TITLE
[a11y] Step content isn't read by screen reader + step buttons not labeled

### ISSUE THREAD

**Issue by quuu** [elrumordelaluz/reactour#278]
Was testing out reactour with MacOS Catalina screen reader voice over on. The steps are fine when navDotAriaLabel is specified, but moving forward and backward arrows are simply labeled as "button". 

The content is also not selectable/ readable. Maybe it's missing an aria label?

**Comment by elrumordelaluz** [comment]
Due to inactivity I am closing this _issue_, but feel free to re-open or create a new one in any case.
